x-pack/filebeat/input/cel/input_test.go

```diff
@@ -1667,7 +1667,7 @@ func TestInput(t *testing.T) {
 				t.Fatalf("unexpected error running test: %v", err)
 			}
 
-			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
I don't think this is the cause here. We do get a response in time, but it is incorrect. The GET request succeeds (in that it returns without error), but the unmarshalling of the body fails due to incompleteness (we don't know how much is missing). Making the timeout longer will not change any of this.

I have investigated the issue and I have a perfect reproducer. I will send a fix.